nixos/support.tt

```diff
@@ -11,6 +11,8 @@
     the list of configuration options by running <tt>man
     configuration.nix</tt>. You can also <a href="options.html">search
     the list of options online</a>.</p>
+
+    <p>There is also <a href="[%nixosManual%]/unstable/">a version for the <strong>nixos-unstable</strong> channel</a>.</p>
```

Comment:
Most of the file is soft-wrapped at around 75 chars. One simple newline would be welcome (imo).

```diff
diff --git a/nixos/support.tt b/nixos/support.tt
index b6b6763..5ccb27d 100644
--- a/nixos/support.tt
+++ b/nixos/support.tt
@@ -12,7 +12,8 @@
     configuration.nix</tt>. You can also <a href="options.html">search
     the list of options online</a>.</p>
 
-    <p>There is also <a href="[%nixosManual%]/unstable/">a version for the <strong>nixos-unstable</strong> channel</a>.</p>
+    <p>There is also <a href="[%nixosManual%]/unstable/">a version for the
+	<strong>nixos-unstable</strong> channel</a>.</p>
   </div>
 
   <div class="span4" id="discourse">
```